lib/commands/performance.js

```diff
@@ -188,6 +189,12 @@ commands.mobileStopPerfRecord = async function (opts = {}) {
                       `You can use 'instruments -s' command to see the list of all available profiles.`);
   }
 
+  if (!_.isEmpty(traceLogPath)) {
+    log.info(`Found perf trace record '${localPath}'. Move to path '${traceLogPath}'.`);
+    await fs.rename(localPath, traceLogPath);
```

Comment:
1. Using `fs.rename` can move the `.trace` document.  
2. The `Appium` will  crash when encoding some extremely large `.trace` documents. So I make this pull request instead of uploading `.trace` document to other servers. I don't think we still need to return its base64-representation. We just need to move the `.trace` directory to a new path with the `traceLogPath` setting.

Reply:
It will only work if the destination folder is under the same parent. Check https://stackoverflow.com/questions/26291647/move-all-files-in-directory-to-parent-with-node-js

Reply:
Regarding p2 - zipping is performed with `zip` tool, so it won't use node's process memory. The resources are only used if we try to base64-encode the result and send it as a response. That step could be skipped though.

Reply:
ok, I will zip the directory.